pkg/kademlia/kademlia.go

```diff
@@ -245,6 +246,14 @@ func (k *Kademlia) FetchInfo(ctx context.Context, node pb.Node) (*pb.InfoRespons
 	return info, nil
 }
 
+// FetchLocalGraph returns the routing table as a dot graph
+func (k *Kademlia) FetchLocalGraph() []byte {
```

Comment:
Kademlia having a method to draw it as a graph feels wrong to me. There are many ways to visualize kademlia and attaching all of those to `Kademlia` itself ends up collecting different concerns, however it helps to have a general concern of "get me the full routing table" and the client decides how to visualize it.